doc/modules/ROOT/pages/index.adoc

```diff
@@ -0,0 +1,320 @@
+
+[source, shell]
+----
+$ helm repo add prometheus-community https://prometheus-community.github.io/helm-charts
+"prometheus-community" has been added to your repositories
+
+$ kubectl create ns monitoring
+namespace/monitoring created
+
+$ helm install prometheus prometheus-community/prometheus --namespace=monitoring
+----
+
+Before installing Prometheus Adapter let's take a look at the content of `hazelcast-custom-metrics-hpa.yaml`
+
+[source, yaml]
+----
+rules:
+  default: true
+  custom:
+  - seriesQuery: '{__name__=~"jvm_memory_bytes_(used|max)",area="heap",kubernetes_name=~"hazelcast.*"}'
+    seriesFilters:
+    - is: ^jvm_memory_bytes_(used|max)$
+    resources:
+      overrides:
+        kubernetes_pod_name: {resource: "pod"}
+        kubernetes_namespace: {resource: "namespace"}
+        kubernetes_name: {resource: "service"}
+    name:
+      matches: ^jvm_memory_bytes_(used|max)$
+      as: "on_heap_ratio"
+    metricsQuery: max(jvm_memory_bytes_used{<<.LabelMatchers>>}/jvm_memory_bytes_max{<<.LabelMatchers>>}) by (<<.GroupBy>>)
+prometheus:
+  url: http://prometheus-server # make sure the url is correct
+  port: 80
+----
+
+This configuration will be passed to the helm chart while deploying Prometheus Adapter but let's just go through each part before doing that. The config basically tells Prometheus Adapter:
+
+* query only `jvm_memory_bytes_used` and `jvm_memory_bytes_max`
+* assign `kubernetes_*` based labels to resources to be able to query via REST URLs like `/apis/custom.metrics.k8s.io/v1beta1/namespaces/default/services/*/on_heap_ratio`
+* give a new, easier name (`on_heap_ratio`) to the metric that we expose via custom metrics adapter
+* select max value out of all series provided by all PODs
+
+This example uses `max` function while creating `metricsQuery`, but you can basically use some other https://prometheus.io/docs/prometheus/latest/querying/operators/#aggregation-operators[aggregation operator] like `avg` in your own configuration. If you saved the file above, you can create a prometheus adapter based on that configuration.
+
+To install Prometheus Adapter run the following command:
+
+[source, shell]
+----
+$ helm install prometheus-adpter prometheus-community/prometheus-adapter --namespace=monitoring -f prometheus-adapter-values.yaml
+----
+
+=== Install Metrics Enabled Hazelcast Cluster
+
+Let's install a new 3 members hazelcast cluster with metrics enabled. Each hazelcast member container in this new deployment will expose their own metrics data under `/metrics` endpoint. This endpoint exposes metrics in Prometheus format because each hazelcast container is started with Prometheus https://github.com/prometheus/jmx_exporter[JMX Exporter]. This is a feature provided by https://github.com/hazelcast/hazelcast-docker[Hazelcast Docker] Image. We also set `resources.limits.memory=512Mi` which sets each hazelcast member JVM max heap size to `128Mi`. JVM by default grabs 25% of available memory as max heap size.
```

Comment:
nit
```suggestion
Let's install a new 3 members Hazelcast cluster with metrics enabled. Each Hazelcast member container in this new deployment will expose their own metrics data under `/metrics` endpoint. This endpoint exposes metrics in Prometheus format because each Hazelcast container is started with Prometheus https://github.com/prometheus/jmx_exporter[JMX Exporter]. This is a feature provided by https://github.com/hazelcast/hazelcast-docker[Hazelcast Docker] Image. We also set `resources.limits.memory=512Mi` which sets each Hazelcast member JVM max heap size to `128Mi`. JVM by default grabs 25% of available memory as max heap size.
```